code/__DEFINES/dna.dm

```diff
@@ -41,18 +41,24 @@
 #define DNA_UI_TAIL_MARK_R	26
 #define DNA_UI_TAIL_MARK_G	27
 #define DNA_UI_TAIL_MARK_B	28
-#define DNA_UI_EYES_R		29
-#define DNA_UI_EYES_G		30
-#define DNA_UI_EYES_B		31
-#define DNA_UI_GENDER		32
-#define DNA_UI_BEARD_STYLE	33
-#define DNA_UI_HAIR_STYLE	34
-#define DNA_UI_HACC_STYLE	35
-#define DNA_UI_BACC_STYLE	36
-#define DNA_UI_HEAD_MARK_STYLE	37
-#define DNA_UI_BODY_MARK_STYLE	38
-#define DNA_UI_TAIL_MARK_STYLE	39
-#define DNA_UI_LENGTH		40 // Update this when you add something, or you WILL break shit.
+#define DNA_UI_GRADIENT_R	29
```

Comment:
I think these should be DNA_UI_HAIR_GRADIENT_..., because otherwise they could be confused a gradient for something else